test/json/json.test.js

```diff
@@ -1,23 +1,39 @@
 'use strict';
-const { join } = require('path');
 const { run } = require('../utils/test-utils');
-const webpack = require('webpack');
 
 describe('json flag', () => {
-    it.skip('should match the snapshot of --json command', async () => {
-        const { stdout } = run(__dirname, [__dirname, '--json']);
-        const jsonstdout = JSON.parse(stdout);
-        const compiler = await webpack({
-            entry: './index.js',
-            output: {
-                filename: 'main.js',
-                path: join(__dirname, 'bin'),
-            },
-        });
-        compiler.run((err, stats) => {
-            expect(err).toBeFalsy();
-            const webpackStats = stats.toJson({ json: true });
-            expect(jsonstdout).toEqual(webpackStats);
```

Comment:
This was invalid from the start, since 2 compilations can never have same hashes and times are also different in most cases, so this should always fail ideally.

Reply:
Yes, you are right!